packages/wallet/backend/src/paymentPointer/controller.ts

```diff
@@ -109,4 +114,30 @@ export class PaymentPointerController implements IPaymentPointerController {
       next(e)
     }
   }
+
+  registerKey = async (
+    req: Request,
+    res: CustomResponse<KeyPair>,
+    next: NextFunction
+  ) => {
+    try {
+      const userId = req.session.user.id
+      const { accountId, paymentPointerId } = req.params
+
+      const { privateKey, publicKey } =
+        await this.deps.paymentPointerService.registerKey(
+          userId,
+          accountId,
+          paymentPointerId
+        )
+
+      res.status(200).json({
+        success: true,
+        message: 'Public key is successfully registered',
+        data: { privateKey, publicKey }
```

Comment:
Maybe keyId might be useful to return as well? Otherwise, if the user can only see a list of keyids, without knowing when they were created, they wouldn't know which id corresponds to which key